bin/bare.c

```diff
@@ -18,6 +22,11 @@ main (int argc, char *argv[]) {
   err = bare_setup(uv_default_loop(), platform, NULL, argc, argv, NULL, &bare);
   assert(err == 0);
 
+#if BARE_USE_SYSTEM_LOG
+  err = log_open("bare", 0);
+  assert(err == 0);
+#endif
+
```

Comment:
Let's move this to the very top of `main()` so other bits can also use the log in the future.